tensorflow/python/estimator/canned/boosted_trees_test.py

```diff
@@ -544,6 +556,439 @@ def testTrainEvaluateAndPredictWithOnlyIndicatorColumn(self):
     self.assertEqual(1, ensemble.trees[0].nodes[0].bucketized_split.feature_id)
     self.assertEqual(0, ensemble.trees[0].nodes[0].bucketized_split.threshold)
 
+  def testExperimentalFeatureImportancesWithTraining(self):
```

Comment:
may be WithTrainedEnsemble

Reply:
Actually do we need this test now (since u added another one that replaces the tree ensemble?

Reply:
The case is special because it uses real ensemble after training, while those tests below all use fake ensemble. I think it's necessary to test.